tests/Support/SupportArrTest.php

```diff
@@ -1112,4 +1112,12 @@ public function testPrependKeysWith()
             ],
         ], Arr::prependKeysWith($array, 'test.'));
     }
+
+    public function testCoalesce()
+    {
+        $this->assertEquals('W3Schools.com', Arr::coalesce([null, null, null, 'W3Schools.com', null, 'Example.com']));
+        $this->assertNotNull(Arr::coalesce([null, null, null, new \DateTime()]));
+        $this->assertNull(Arr::coalesce(null));
+        $this->assertNull(Arr::coalesce([]));
```

Comment:
Add one test with array of one null value
`[null]`

And what will happen if string keyed array will be passed?